opencraft/settings.py

```diff
@@ -596,3 +600,22 @@
 AWS_ACCESS_KEY_ID = env('AWS_ACCESS_KEY_ID', default=None)
 AWS_S3_BUCKET_PREFIX = env('S3_BUCKET_PREFIX', default='ocim')
 AWS_IAM_USER_PREFIX = env('IAM_USER_PREFIX', default='ocim')
+
+# Consul ######################################################################
+
+# The encryption key used to gossip in a Consul cluster.
+CONSUL_ENCRYPT = env('CONSUL_ENCRYPT', default='')
+
+# The list of server agents in the Consul cluster.
+CONSUL_SERVERS = env.list('CONSUL_SERVERS', default=[])
```

Comment:
Same here – we need to make sure the Ansible code emits a comma-separated list instead of the Python representation we are currently writing.  Alternatively, we can consider using JSON instead.